README.md

```diff
@@ -1 +1,49 @@
-# node-hello-world-lab
\ No newline at end of file
+# Node Hello World
+
+## Objectives
+
+1. Implement Hello World in Node
+1. Run the script from the terminal
+
+
+## Introduction
+
+In this lab, you'll practice how to run Node scripts from command line. In addition, you'll develop your first Node application—Hello World, the traditional first app in any language.
+
+## Instructions
+
+1. Create file `hello-world.js` and open it in your favorite editor
+2. Implement an expression (function with `return`) which returns "Hello World"
+3. Export your expression (function with `return`) with `module.exports = ...`
+5. Install testing dependencies with `$ npm install` (`$` is a symbol which tells you that this is a command line command). You need to do it just once for each lab.
```

Comment:
for consistency's sake with the rest of Flatiron's curriculum, please remove all `$`